module/userCourses/helper.js

```
@@ -0,0 +1,211 @@
+			// Mandatory Field Checks
+			if (!userCoursesData.userId || !userCoursesData.status || !userCoursesData.entityId) {
+				return {
+					success: false,
+					status: HTTP_STATUS_CODE.bad_request.status,
+					message: CONSTANTS.apiResponses.MISSING_SOLUTIONID_USERID_STATUS,
+				}
+			}
+
+			// get the userCourses document
+			let courseDocument = await userCoursesQueries.userCoursesDocuments({
+				solutionId: new ObjectId(userCoursesData.entityId),
```

ESLint

[error] 'ObjectId' is not defined.

(no-undef)